test/surface/wac-allow.test.ts

```diff
@@ -192,5 +192,15 @@ describe('From default', () => {
       const result = await fetch(`${testFolderUrl}3/publicReadBobWrite/test.txt`);
       expect(sortWac(result.headers.get('WAC-Allow'))).toEqual(sortWac('user="read append",public="read append"'));
     });
+    it(`Shows the Link header containing the aclDocUrl to Alice`, async () => {
+      const result = await solidLogicBob.fetch(`${testFolderUrl}3/publicReadBobWrite/test.txt`);
+      const aclDocUrl = await solidLogicAlice.findAclDocUrl(`${testFolderUrl}3/publicReadBobWrite/test.txt`);
+      expect(result.headers.get('Link')).toContain(aclDocUrl);
+    });
+    it(`Does not show a Link header containing the aclDocUrl to the public`, async () => {
```

Comment:
OK to add the code for this test but please add it with `it.skip` and add a comment 'DISPUTED: See https://github.com/solid/specification/pull/248

Reply:
Marking these as skipped with the comment and link to the dispute.